docs/criteria/accelerators/beta.md

```diff
@@ -87,23 +85,41 @@ Indexes are not required for test coverage, but can be introduced if required fo
   - [ ] Struct
   - [ ] Decimal128/Decimal256
 - [ ] Integration tests to cover "On Conflict" behaviors.
-- [ ] A reproducible test script that can load a variety of datasets into accelerators in both file/embedded memory mode.
```

Comment:
>  A reproducible test script
What is the desired mechanism we want to for us, and open source contributors to run these tests?

Reply:
I don't understand your comment here. This is a diff for removal too? The script is only to help load data for SF10 and SF100 TPCH/TPCDS

Reply:
Sorry, I meant, don't we still want accelerators to have these reproducible test script? Why did we remove the requirement?